### PR TITLE
Move sample code to example directory

### PR DESCRIPTION
**Pull request by MrMinimal**
For convenience I have moved the sample code to an actual example.
It can be run using a built-in `cargo` command, which I have added to the README.

These lines of the example don't compile with the latest simconnect version anymore and are therefore commented out:
https://github.com/MrMinimal/simconnect-rust/blob/ebef846aa453c5daa6821742fa56084818ad5c6f/examples/connection/main.rs#L54

Edits welcome 😊

### CODE REVIEW

**Comment by BitsAndDroids** [comment]
I've opened a pr as well that required an extra example. I've incorporated your suggestion but also fixed the broken example. 
(if this repo is still maintained that is)
https://github.com/Sequal32/simconnect-rust/pull/10

**Comment by MrMinimal** [comment]
@BitsAndDroids Thank you so much, I had a look at your PR and it should greatly improve the usability of this library.

File: examples/connection/main.rs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        0,
+        0,
+    ); //request_id, define_id, object_id (user), period, falgs, origin, interval, limit - tells simconnect to send data for the defined id and on the user aircraft
+
+    loop {
+        match conn.get_next_message() {
+            Ok(simconnect::DispatchResult::SimobjectData(data)) => unsafe {
+                match data.dwDefineID {
+                    0 => {
+                        // let sim_data: DataStruct = transmute_copy(&data.dwData);
```
Comment (Sequal32) [inline examples/connection/main.rs:54]: I don't think these should be commented out

File: Readme.md [hunk 2]
```diff
@@ -1,57 +1,25 @@
 # SimConnect Bindings for Rust
-## Using
+## Usage
 Add this to your `Cargo.toml`
 ```toml
 [dependencies]
 simconnect = "0.2.0"
 ```
+*This crate is in its early stages and API breaking changes can be pushed at any moment. It's recommended to use the exact version "0.X.X"*
```
Comment (Sequal32) [inline Readme.md:8]: This was removed in 6da1fc65d9aca4f753fcce7004f6232a919e7452
```suggestion

```

**Review by Sequal32 — CHANGES_REQUESTED** [review]
Sorry just got around to looking at this now, and thanks so much for the contribution! These changes will make the Repo easier to demo with the example binary now. 

Just a couple of changes.

**Comment by Sequal32** [comment]
Sorry just got around to looking at this now, and thanks for your contribution! I'll close this in favor of @BitsAndDroids 's PR #10.

**Comment by MrMinimal** [comment]
@Sequal32 Cheers and happy holidays, really appreciate your project!

**Pull request closed without merging**